### PR TITLE
Feat/benefits band

### PR DESCRIPTION
**Pull request by emerson-pereira**
### Related Ticket(s)

Addresses #345 

### Description

Develop Simple Benefits Band React Pattern

### Changelog

**New**

- Simple Benefits Band Pattern

### CODE REVIEW

**Comment by jeffchew** [comment]
The name "Simple Benefits Band"....should we remove "Band"?

cc: @wonilsuhibm @emerson-pereira @larahanlon2

**Comment by jeffchew** [comment]
Shouldn't the CTAs be aligned with each other? @wonilsuhibm @larahanlon2 @oliviaflory ?

Screen Shot 2020-01-07 at 4 45 37 PM

**Comment by oliviaflory** [comment]
From the spec file it looks like we have the CTA at a set space from the end of the body text, so all CTA positions will vary. 

But it does look like the CTAs are just a little too far apart from the body text, currently there's 32px between body / CTA and but in the spec file I see a mix of either 22px or 24px. 

@wonilsuhibm I thought we decided not to go with 22px because it's not a mini unit or in the layout spacers, can you confirm?

**Comment by larahanlon2** [comment]
> The name "Simple Benefits Band"....should we remove "Band"?
> 
> cc: @wonilsuhibm @emerson-pereira @larahanlon2

Yes, we should remove the word **Band** from all of our patterns. Also, the name "Simple Benefits" might change because we're refining the names (and naming conventions) for all patterns.

**Comment by emerson-pereira** [comment]
> > The name "Simple Benefits Band"....should we remove "Band"?
> > cc: @wonilsuhibm @emerson-pereira @larahanlon2
> 
> Yes, we should remove the word **Band** from all of our patterns. Also, the name "Simple Benefits" might change because we're refining the names (and naming conventions) for all patterns.

Removed the word "band"

**Comment by emerson-pereira** [comment]
> > Shouldn't the CTAs be aligned with each other?
> 
> From the spec file it looks like we have the CTA at a set space from the end of the body text, so all CTA positions will vary.
> 
> But it does look like the CTAs are just a little too far apart from the body text, currently there's 32px between body / CTA and but in the spec file I see a mix of either 22px or 24px.
> 
> @wonilsuhibm I thought we decided not to go with 22px because it's not a mini unit or in the layout spacers, can you confirm?

Waiting conformation on the 22px thing. Meanwhile I updated spacing to 24px.

File: packages/styles/scss/patterns/simplebenefits/_simplebenefits-item.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      }
+    }
+
+    &__title {
+      margin-bottom: $carbon--layout-03;
+      max-width: 80%;
+      @include carbon--type-style('expressive-heading-03');
+    }
+
+    &__devider {
```
Comment (jeffchew) [inline packages/styles/scss/patterns/simplebenefits/_simplebenefits-item.scss]: @emerson-pereira minor thing, `divider` is spelled wrong here...

Comment (emerson-pereira) [reply packages/styles/scss/patterns/simplebenefits/_simplebenefits-item.scss]: Fixed!

**Pull request merged**